pkg/kubeapiserver/admission/initializer.go

```diff
@@ -62,7 +62,7 @@ type WantsQuotaConfiguration interface {
 // WantsServiceResolver defines a fuction that accepts a ServiceResolver for
 // admission plugins that need to make calls to services.
 type WantsServiceResolver interface {
-	SetServiceResolver(webhook.ServiceResolver)
+	SetServiceResolver(options.ServiceResolver)
```

Comment:
Can we move the interface and the code applying it to plugins to k8s.io/apiserver as well? This split does not make much sense now.

Reply:
I agree they should go there, but not in this PR.